templates/github/.ci/ansible/settings.py.j2.copy

```diff
@@ -8,7 +8,7 @@ TOKEN_SIGNATURE_ALGORITHM = "ES256"
 CACHE_ENABLED = True
 REDIS_HOST = "localhost"
 REDIS_PORT = 6379
-TELEMETRY = False
+ANALYTICS = False
```

Comment:
Thanks. The reason is that the dev.analytics... will still happily sum up all systems, as we set the `..._MIN_AGE=0` there.